docs/one-path-reachability-proofs.md

```diff
@@ -0,0 +1,174 @@
+Using a rule to advance a one-path reachability proof
+=====================================================
+
+This document describes how to transform a given a one-step reachability goal
+by advancing it using a given rewrite axiom.
+
+This document *does not describe* how to select an axiom to be used for
+advancing the goal.
+
+
+Problem statement
+-----------------
+
+Consider the following "one-path" reachability goal
+
+```
+∀ X . φ(X) → ◆ ∃ Y . ψ(X, Y)
+```
+where `X` and `Y` are sets of variables and ◆ denotes "strong-eventually"
+
+Consider also the axiom
+
+`∀ Z . α(Z) → ⚪ β(Z)`.
+
+where ⚪  denotes "strong-next".
```

Comment:
The `applying-axioms.md` document uses a full circle for strong-next, why not use a consistent notation?